schemas/2015-10-31/Microsoft.Automation.json

```diff
@@ -740,6 +744,66 @@
       ],
       "description": "Microsoft.Automation/automationAccounts/variables"
     },
+    "automationAccounts_watchers": {
```

Comment:
>automationAccounts_watchers [](start = 5, length = 27)

add the new type in deploymentTemplate.json?

Reply:
@TianoMS Added to deploymentTemplate.json and the tests for it.